cmpserver/plugin/plugin.go

```diff
@@ -259,13 +259,13 @@ func (s *Service) generateManifest(ctx context.Context, appDir string, envEntrie
 
 	env := append(os.Environ(), environ(envEntries)...)
 	if len(config.Spec.Init.Command) > 0 {
-		_, err := runCommand(ctx, config.Spec.Init, appDir, env)
+		_, err := runCommand(ctx, config.Spec.Init, "Plugin init returned zero output (this is ok)", appDir, env)
```

Comment:
I think the messages should not be passed literally to the `runCommand` function. IMHO, a better solution would be something like an "empty output severity" (e.g. an enum), which the `runCommand` function will then handle.

For example (just some pseudo code):

```go
type emptyOutputCondition int
const (
   emptyOutputOK emptyOutputCondition = 0,
   emptyOutputWarn,
   emptyOutputUnsupported
)

// Then, prototype of runCommand function looks like:

func runCommand(ctx context.Context, command Command, onEmpty emptyOutputCondition, path string, env []string) (string, error) 

// and within runCommand the output is produced like so:

if len(output) == 0 {
   switch onEmpty {
   case emptyOutputOK:
      log.Info("Plugin init returned zero output (this is ok)")
   case emptyOutputWarn:
      log.Warn("Plugin command returned zero output (this is likely problematic)")
   // etc etc
   }
}

This will help with a) consistent output, even if reused, b) shorter function calls, c) less repetition

WDYT?